lang/python/python/files/python-config.in

```diff
@@ -0,0 +1,81 @@
+#!@EXENAME@
```

Comment:
i'm a bit curios about the use-case on this one;
i.e what's the difference between this one and Python's;

i'll take a look
i'm a bit hesitant right now about this one

Reply:
Comparing with the [default script from Python 2](https://github.com/python/cpython/blob/2.7/Misc/python-config.in):

* In both, `@EXENAME@` will be replaced with host Python (for this local version, this is done in `Build/InstallDev`)
* After `from distutils import sysconfig`, replace host Python build info with target Python data (loaded from `$(STAGING_DIR)/usr/lib/python2.7-openwrt/_sysconfigdatatarget.py`)
* In the print statements at the end of the script, replace host prefix (`$(STAGING_DIR_HOST)`) with target prefix (`$(STAGING_DIR)/usr`) (the prefix isn't completely replaced when target Python data is loaded earlier)